### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by willfuchs-RL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # pr-workflow-test
```
Comment (willfuchs-RL) [inline README.md:1]: stuff here

**Comment by willfuchs-RL** [comment]
comment

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 A test repo to mess around with the draft PR workflow
 
-Testing actions on updgrade PR from draft to full on PR
```
Comment (willfuchs-RL) [inline README.md:4]: comment 1

Comment (willfuchs-RL) [reply README.md:4]: reply to comment

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
 # pr-workflow-test
 A test repo to mess around with the draft PR workflow
```
Comment (haleyraizes-RL) [inline README.md:3]: You could add a period at the end

File: README.md (hunk 2 above)
Comment (haleyraizes-RL) [inline README.md:4]: Hello Will

File: README.md (hunk 3 above)
Comment (willfuchs-RL) [reply README.md:3]: Sounds like a good idea to me.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
```
Comment (haleyraizes-RL) [inline README.md:2]: This is a test for the status of a review with requested changes

**Pull request closed without merging**